### DISCUSSION TITLE
url of a dir in md aren't work well

### DISCUSSION THREAD

**Discussion by Puellaquae** [community/community#7758] [Mobile]
In md file, url link to a dir in the repository cannot jump correct. In website a url, such as `./dir`, will be replaced by `./blob/master/dir` and redirect to `./tree/master/dir` when open, while in the app it just open `./blob/master/dir` and shows nothing. But, the url link to file can work well.

**Comment by brianlovin** [comment]
Hey @Puellaquae I _think_ I followed your report, but could you provide a working public example so that we can try to reproduce this? Thank you!

**Reply by Puellaquae** [reply]
Hi, here is a my repo [Puellaquae/CompilerLabs](https://github.com/Puellaquae/CompilerLabs). There have some url to dir in readme which can produce this problem.

**Reply by brianlovin** [reply]
I'm not sure I'm able to reproduce the bug — when I view that repository on iOS, I can tap the links in the README to navigate to the correct place in the code browser. Do you have a video or more details about what's happening on your side?

**Reply by Puellaquae** [reply]
I don't have iOS device, maybe it only happens in Android. I capture a video.
https://user-images.githubusercontent.com/22560343/148321620-10027042-f293-45e4-aa7b-c4d7af9a28bb.mp4

**Reply by brianlovin** [reply]
That video isn't working for me, are you able to play it? I will investigate on Android though.

**Reply by brianlovin** [reply]
I can reproduce the bug on Android, I'll flag this for the team. Thanks for helping here @Puellaquae 😊
